distributions/standalone/pom.xml

```diff
@@ -38,6 +38,33 @@
                 </executions>
             </plugin>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy-plugins</id>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <configuration>
+                            <skip>${skipDistributions}</skip>
+                            <useBaseVersion>true</useBaseVersion>
```

Comment:
If you set `outputDirectory` as `${project.build.directory}/plugins` then you avoid modifying the `assembly.xml`